common/views/components/BasePage/BookPage.js

```diff
@@ -45,24 +43,25 @@ const BookPage = ({ book }: Props) => {
   };
   // $FlowFixMe
   const FeaturedMedia = book.cover && <UiImage tasl={tasl} extraClasses='margin-h-auto width-auto max-height-70vh' {...book.cover} />;
-  const Header = (<BaseHeader
-    title={book.title || ''}
-    Background={<HeaderBackground hasWobblyEdge={true} />}
-    Breadcrumb={<Breadcrumb items={[{
-      text: 'Book',
+  const breadcrumbs = {
+    items: [{
+      text: 'Books',
       url: '/books'
-    }]} />}
-    DateInfo={null}
-    Description={
+    }]
+  };
+  const NewHeader = <PageHeader
```

Comment:
Why is this NewHeader and not just Header?

Reply:
There's something already in Header, which seems to be `SiteHeader`

Reply:
Ignore me, you changed it in the next commit :-)